tripy/tripy/frontend/shape.py

```diff
@@ -149,6 +177,38 @@ def __radd__(self, other):
             other = Shape(other)
         return concatenate([other, self], 0)
 
+    # multiplication for shapes is tiling, not elementwise multiplication
+
+    def __mul__(self, other):
```

Comment:
Are there calls to this that were relying on the old behavior of `__mul__`?

Reply:
I don't believe so, other than those in the tests updated here. I've searched for all appearances of ` * ` in the Tripy codebase and they don't seem to be used with shapes. The CI will tell us in either case.

As with our handling of `+`, this violates the Liskov substitution principle, so that's a hazard of inheritance here 😅